OpenSSL missing on Windows targets

## Description
General problem: OpenSSL libraries are not available on GitHub hosted runners for Windows. They are also not installed by the setup-ros action.

Specific problem: I'm trying to build an overlay workspace for generating messages with a custom IDL generator. This requires `rosidl_default_generators` which ultimately pulls in `rosidl_typesupport_fastrtps_cConfig.cmake` and `fastrtps-config.cmake` which, for the binary distribution, has a hard dependency on SSL.

### Expected Behavior
OpenSSL is a ROS2 dependency, so I would expect this action to properly install and configure OpenSSL and required environment variables.

### Actual Behavior
Output such as:
```
CMake Error at C:/Program Files/CMake/share/cmake-3.14/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
  system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY
  OPENSSL_INCLUDE_DIR)
```

## To Reproduce
I've attached an overlay repos file (ros2_message_overlay.txt) which fails to build in a GitHub Action due to missing OpenSSL.

Alternatively, I've also produced a similar output building an overlay workspace which contains nothing but the `unique_identifier_msgs` package.

[ros2_message_overlay.txt](https://github.com/ros-tooling/setup-ros/files/4567499/ros2_message_overlay.txt)

## System (please complete the following information)
 - OS: Windows 10
 - ROS 2 Distro: Eloquent

--- a/src/package_manager/chocolatey.ts
+++ b/src/package_manager/chocolatey.ts
@@ -7,7 +7,13 @@ const chocoCommandLine: string[] = [
 	"--yes",
 ];
 
-const chocoDependencies: string[] = ["cppcheck", "wget", "7zip", "lcov"];
+const chocoDependencies: string[] = [
+	"cppcheck",
+	"wget",
+	"7zip",
+	"lcov",
+	"openssl",
+];
 
 const ros2ChocolateyPackagesUrl: string[] = [
 	"https://github.com/ros2/choco-packages/releases/download/2019-10-24/asio.1.12.1.nupkg",

